src/ui/dialogs/contactBlock.ts

```diff
@@ -0,0 +1,71 @@
+import Dialog from '../Dialog'
+import Translation from '../../util/Translation'
+import Client from '../../Client'
+import BlockingCommandPlugin from '../../plugins/BlockingCommandPlugin'
+
+let contactBlockList = require('../../../template/contactBlock.hbs');
+
+export default function() {
+   let content = contactBlockList({});
+
+   let dialog = new Dialog(content);
+
+   let account = Client.getAccountManager().getAccounts()[0];
+   let connection = account.getConnection();
+   let items;
+
+   BlockingCommandPlugin.getInstance().getBlocklist()
```

Comment:
We need probably a `getPlugin` method in `PluginRepository` similar to `getEncryptionPlugin` for this.

Reply:
Yes that would be nice... I also have this problem in other PRs too

Reply:
i have added the getPlugin() to PluginRepository ... is working fine...